molecule/commands.py

```diff
@@ -159,6 +159,8 @@ def converge(self, idempotent=False, create_instances=True, create_inventory=Tru
             kwargs.pop('_err', None)
             kwargs['_env']['ANSIBLE_NOCOLOR'] = 'true'
             kwargs['_env']['ANSIBLE_FORCE_COLOR'] = 'false'
+            kwargs['_env']['ANSIBLE_CALLBACK_PLUGINS'] = os.path.join(
+                sys.prefix, 'share/molecule/ansible/plugins/callback/idempotence')
```

Comment:
That's the point. The plugin really has only one usage, which is to make it easy to identify the "changed" tasks from the playbook output. If the plugin is replaced, the idempotence fucntion may not work as expected. 

As a compromise I could read from the config file and if no value is specified, I would set that one as default. But again, I'm not sure this should be a value that should be configured.